docs/git.rst

```diff
@@ -560,6 +560,15 @@ Again this applies:
 http://www.osnews.com/story/19266/WTFs_m
 
 
+The review-change-push-review cycle
+-----------------------------------
+
+After you have made a pull request, you should get a review via the github interface. After the review you will frequently have to do some changes of your PR, add a missing changelog entry, and so on.
+When the changes are done, rebase your branch again, squash commits, all as described above.
+Finally you should force push your feature branch to gitlab. Only force push on your own feature branches! Never on branches shared with other people.
```

Comment:
GitHub instead of gitlab.
Also please use the correct ``GitHub`` camel case wording instead ``github`` everywhere.